libraries/anaconda3-buildtime/Dockerfile

```diff
@@ -1 +1,3 @@
 COPY --chown=0:0 anaconda3-buildtime/context/build /usr/local/bin/algorithmia-build
+RUN conda create --prefix /tmp/conda_environment && \
+    chmod --recursive ugo+rw /tmp/conda_environment
```

Comment:
Should we change to `USER algo` before the `conda create` so permissions will be set for the algo users to read and write to that directory?

Reply:
use algo isn't defined at this point, that happens inside of build.j2 & compile.j2, would creating the user in advance in this instance mess anything up? I'm down with chowning to user algo:algo

Reply:
I presume you're referencing [Dockerfile.builder.j2](https://github.com/algorithmiaio/langpacks/blob/master/languages/Dockerfile.builder.j2), in which case I believe that `anaconda3-buildtime` would be templated in as a package [on line 17](https://github.com/algorithmiaio/langpacks/blob/master/languages/Dockerfile.builder.j2#L17), which is after the `algo` user is [defined above](https://github.com/algorithmiaio/langpacks/blob/master/languages/Dockerfile.builder.j2#L11). So, it should be defined at this point.